__tests__/plants/getPlantsOfUserWithPagination.mjs

```diff
@@ -0,0 +1,40 @@
+import { describe, it, expect } from "@jest/globals";
+
+import * as client from "../../src/clients/clients.mjs";
+import { initializeDatabase } from "../../src/database/initializeDatabase.mjs";
+import { clearDatabase } from "../../src/database/clearDatabase.mjs";
+import { errorMessages } from "../../src/errors/errors.mjs";
+
+describe("Get plants of user", () => {
+  beforeAll(async () => {
+    await initializeDatabase("Plants");
+    await initializeDatabase("Users");
+  });
+
+  afterAll(async () => {
+    await clearDatabase("Plants");
+    await clearDatabase("Users");
+  });
+
+  it(`Should return correct message when user has no more plants`, async () => {
+    const response = await client.getAllPlantsOfUserWithPagination("jest_user_1", "12");
+    const result = JSON.parse(response.text);
+
+    expect(result.message).toEqual(errorMessages.userHasNoPlants);
```

Comment:
Is deze test nodig? Kan er een pagina nog zijn, waar er geen planten in staan?

Reply:
Nee, normaal ni, maar ik wouw gewoon de melding nog eens testen voor de zekerheid